packages/dx-react-chart/docs/guides/getting-started.md

```diff
@@ -52,7 +52,7 @@ React Chart uses the latest web platform standards and does not support older br
 
 ## Using the Chart Component
 
-The Chart component's functionality is implemented in plugins. You need a [series](series.md) plugin and the `ArgumentAxis`, `ValuesAxis`, and `Scale` plugins to draw a simple chart. The following code shows how to configure a simple line chart:
+The Chart component's functionality is implemented in plugins. To draw a simple chart, the `ArgumentAxis` and `ValueAxis` plugins and one of the [series](series.md) plugins. The following code shows how to configure a simple line chart:
```

Comment:
"To draw a simple chart, the `ArgumentAxis` and `ValueAxis` plugins and one of the [series](series.md) plugins." This is an incomplete sentence.